sklearn/model_selection/tests/test_search.py

```diff
@@ -623,25 +647,38 @@ def test_pandas_input():
     for InputFeatureType, TargetType in types:
         # X dataframe, y series
         X_df, y_ser = InputFeatureType(X), TargetType(y)
-        check_df = lambda x: isinstance(x, InputFeatureType)
-        check_series = lambda x: isinstance(x, TargetType)
+
+        def check_df(x):
+            return isinstance(x, InputFeatureType)
+
+        def check_series(x):
+            return isinstance(x, TargetType)
+
         clf = CheckingClassifier(check_X=check_df, check_y=check_series)
 
+        # Default scoring
         grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]})
         grid_search.fit(X_df, y_ser).score(X_df, y_ser)
-        grid_search.predict(X_df)
         assert_true(hasattr(grid_search, "cv_results_"))
 
+        # Multi-metric scoring
```

Comment:
but even single `scoring` hasn't been tested here. I agree that testing pandas with scoring is appropriate, but I don't see why it's the job of this PR.